recipes/sagemaker-studio-dataengineering-extensions/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "sagemaker-studio-dataengineering-extensions" %}
+{% set version = "1.0.5" %}
+{% set python_min = "3.11" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/sagemaker_studio_dataengineering_extensions-{{ version }}.tar.gz
+  sha256: 1ab3c03b8abba899e2d8674950fdd39b926269c909ff3212c467ebbf3dc40fe2
+
+build:
+  noarch: python
+  script: ./src/sagemaker_studio_dataengineering_extensions/scripts/pypi_pre_build.sh && {{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv 
```

Comment:
i'm not sure what this script does, but it feels suspect can you link us to the source of this script so I can see what it does?

Reply:
Sure. 

Basically this script is to change pyproject.toml file to remove some AWS internal hatch plugin, which is required in AWS customized hatch build system but not for public hatch. I'm planing to remove this in the next revision as well.

You could download the source from https://pypi.org/project/sagemaker-studio-dataengineering-extensions/#sagemaker_studio_dataengineering_extensions-1.0.5.tar.gz.